bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/BridgeType.java

```diff
@@ -100,10 +103,10 @@ public BridgeType(ThingTypeUID uid, List<String> supportedBridgeTypeUIDs, String
      * @throws IllegalArgumentException if the UID is null or empty,
      *             or the the meta information is null
      */
-    public BridgeType(ThingTypeUID uid, List<String> supportedBridgeTypeUIDs, String label, String description,
-            boolean listed, List<ChannelDefinition> channelDefinitions,
-            List<ChannelGroupDefinition> channelGroupDefinitions, Map<String, String> properties,
-            URI configDescriptionURI) throws IllegalArgumentException {
+    public BridgeType(@NonNull ThingTypeUID uid, @Nullable List<String> supportedBridgeTypeUIDs, @NonNull String label,
```

Comment:
Didn't we agree on that we only want to state if a parameter is `NonNull` and if if can be `null` we omit the `@Nullable` annotation?